code/apps/Managed Software Center/Managed Software Center/en.lproj/MainMenu.xib

```diff
@@ -439,10 +442,10 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <clipView key="contentView" id="rwq-IA-IvG">
                             <rect key="frame" x="1" y="1" width="512" height="447"/>
-                            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                            <autoresizingMask key="autoresizingMask"/>
                             <subviews>
                                 <tableView focusRingType="none" verticalHuggingPriority="750" allowsExpansionToolTips="YES" columnAutoresizingStyle="lastColumnOnly" alternatingRowBackgroundColors="YES" columnReordering="NO" columnResizing="NO" autosaveColumns="NO" typeSelect="NO" id="UAy-M3-yRr">
-                                    <rect key="frame" x="0.0" y="0.0" width="512" height="17"/>
+                                    <rect key="frame" x="0.0" y="0.0" width="512" height="447"/>
```

Comment:
That is odd. I'll take a look at applying the change to a build without this.

My first guess might have been that it was just the state of the xib view window itself? Since I almost never look at diffs of Xcode projects I'm just guessing.